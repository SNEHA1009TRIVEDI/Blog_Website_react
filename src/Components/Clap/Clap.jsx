import React from "react";
import { isPropertySignature } from "typescript";

export default function Clap({iconPath, clapCount, size = 25}) {
  return (
    <>
    {/* {console.log(props)} */}
    <img
      src={iconPath}
      className="card-actions-icon"
      width={size}
      height={size} 
      alt="clap"
      onClick={() => {clapCount()}}
    />
    </>
  );
}
