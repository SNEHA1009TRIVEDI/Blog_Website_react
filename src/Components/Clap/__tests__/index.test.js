import { render } from "@testing-library/react";
import React from "react";
import  Clap from "../Clap.jsx";

describe("Clap", () => {
  it("should renders without crashing ", () => {
    const { asFragment } = render(<Clap />);
    expect(asFragment()).toMatchSnapshot();
  });
});
