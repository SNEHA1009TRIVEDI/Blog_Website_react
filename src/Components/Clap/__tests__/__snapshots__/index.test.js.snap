// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Clap should renders without crashing  1`] = `
<DocumentFragment>
  <img
    alt="clap"
    class="card-actions-icon"
    height="25"
    width="25"
  />
</DocumentFragment>
`;
